pkg/handler/handler.go

```diff
@@ -141,16 +143,24 @@ func (m *Modifier) updatePodSpec(pod *corev1.Pod, roleName, audience string) []p
 		}
 	}
 
+	tokenFilePath := filepath.Join(m.MountPath, m.tokenName)
+	if val, ok := pod.Spec.NodeSelector["beta.kubernetes.io/os"]; ok && val == "windows" {
```

Comment:
Sorry, one last thing. The [documented node OS label for 1.14](https://v1-14.docs.kubernetes.io/docs/concepts/configuration/assign-pod-node/#interlude-built-in-node-labels) is `kubernetes.io/os`. Can you have this match either `beta.kubernetes.io/os` or `kubernetes.io/os` and update the documentation? Everything else looks great

Reply:
@micahhausler done. Added support+tests for both node selectors.